osmoutils/cli_helpers.go

```diff
@@ -47,3 +49,15 @@ func ParseSdkIntFromString(s string, separator string) ([]sdk.Int, error) {
 	}
 	return parsedInts, nil
 }
+
+// ConditionalPanic checks if expectPanic is true, asserts that sut (system under test)
+// panics. If expectPanic is false, asserts that sut does not panic.
+// returns true if sut panics and false if it does not
+func ConditionalPanic(t *testing.T, expectPanic bool, sut func()) {
```

Comment:
I'm not sure if this is the correct file for such a helper to exist. Do have a place for more general shared testing auxiliary stuff?

Reply:
created a new file called `test_helpers.go` as matt suggested